### PR TITLE
Consolidate some form and button styles

### PR DESCRIPTION
**Pull request by lyzadanger**
In advance of some migration work, remove some unused classes, extract
some color variables and organize.

### CODE REVIEW

File: src/sidebar/templates/annotation.html [hunk 1]
```diff
@@ -37,14 +37,14 @@
       h-branding="accentColor">mire</a>
   </div>
   <!-- Tags -->
-  <div class="annotation-body form-field" ng-if="vm.editing()">
+  <div class="annotation-body" ng-if="vm.editing()">
```
Comment (lyzadanger) [inline src/sidebar/templates/annotation.html]: `.form-field` had extant rules, but they did nothing here as all of them were overridden by more specific rules.

File: src/styles/forms.scss [hunk 2]
```diff
@@ -28,62 +17,3 @@
   max-width: 100%;
   resize: vertical;
 }
-
-.form-actions {
-  @include forms.pie-clearfix;
-  margin-top: 0.75em;
-}
-
-.btn {
```
Comment (lyzadanger) [inline src/styles/forms.scss]: Hard to grep for! But I tried to be thorough and I am confident there are no remaining uses of this class in either annotator or sidebar source.

File: src/styles/mixins/forms.scss [hunk 3]
```diff
@@ -35,63 +35,25 @@
...
-  border-color: #bababa;
-}
-
-@mixin btn-disabled {
-  box-shadow: none;
-  cursor: default;
-  background: #f0f0f0;
-  border-color: #cecece;
-  color: var.$grey-semi;
+  border-color: var.$grey-4;
```
Comment (lyzadanger) [inline src/styles/mixins/forms.scss]: `btn-hover` should be able to go as soon as the new `TagEditor` lands as the existing tag editor is its only use in the app.

File: src/styles/variables.scss [hunk 4]
```diff
@@ -4,6 +4,10 @@
 
 $white: #fff !default;
 
+// Interim color variable for migration purposes.
```
Comment (lyzadanger) [inline src/styles/variables.scss]: I'm not thrilled here to have another grey, but this color of grey is not substitute-able with `$grey-1` as it is quite subtle.

File: src/styles/variables.scss [hunk 5]
```diff
@@ -34,6 +38,9 @@ $black: #000 !default;
 $brand: #bd1c2b;
 $highlight: #58cef4;
 
+$color-focus-outline: #51a7e8;
```
Comment (lyzadanger) [inline src/styles/variables.scss]: I prefixed this color variable name with `$color-`. I'm not married to this naming convention (nor do I feel like we need to have all conventions sewn up at this time; we can adapt as we go).

File: src/styles/forms.scss (hunk 2 above)
Comment (robertknight) [reply src/styles/forms.scss]: I couldn't find any references either with `(?<![A-Za-z_-])btn` as the pattern (match `btn` [not preceded by](https://www.regular-expressions.info/refadv.html) a letter, hyphen or dash)

File: src/styles/variables.scss (hunk 5 above)
Comment (robertknight) [reply src/styles/variables.scss]: We currently have some form fields / buttons using the native focus ring and other inputs using this more Bootstrap-y thing. We need to sort that out. I wonder if we should just go with the native focus ring everywhere. This is fine for the moment.

**Pull request merged**